web-api/src/storage/elastic.rs

```diff
@@ -81,98 +83,76 @@ impl Default for Config {
     }
 }
 
-pub(crate) struct ElasticSearchClient {
-    config: Config,
-    url_to_index: Url,
-    client: reqwest::Client,
-}
-
-impl ElasticSearchClient {
-    pub(crate) fn new(config: Config) -> Result<Self, SetupError> {
```

Comment:
this moved into a setup method of the elastic config (similarly to the postgres code)